eoscostarica/eos-rate
# Improve testing workflow

### GH Issue

#

### Steps to test

1. execute scripts/deploy.sh testing ${public_ip}
1. visit eosrate.io:8080
1. you should see the eos rate page

### Checklist
